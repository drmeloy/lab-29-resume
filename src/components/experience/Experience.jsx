import React from 'react';
import PropTypes from 'prop-types';

const Experience = experience => {
  const experiences = experience.map(item => (
    <li>{item}</li>
  ));

  return(
    <>
      <h3>Experience</h3>
      <ul>{experiences}</ul>
    </>
  );
};

Name.PropTypes = {
  experience: PropTypes.array.isRequired
}

export default Experience;
