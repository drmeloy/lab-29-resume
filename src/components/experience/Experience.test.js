import React from 'react';
import { shallow } from 'enzyme';
import Experience from './Experience.jsx';

describe('Experience component', () => {
  it('renders a Experience component', () => {
    const wrapper = shallow(<Experience experience={[]} />);
    expect(wrapper).toMatchSnapshot();
  })
});
