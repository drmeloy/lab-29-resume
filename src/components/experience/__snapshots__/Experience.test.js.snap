// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Experience component renders a Experience component 1`] = `
<Fragment>
  <h3>
    Experience
  </h3>
  <ul />
</Fragment>
`;
